packages/ag-charts-community/src/util/module.ts

```diff
@@ -90,7 +94,7 @@ export type Module<M extends ModuleInstance = ModuleInstance> =
     | AxisModule
     | AxisOptionModule
     | LegendModule
-    | SeriesModule;
+    | SeriesModule<any>;
```

Comment:
nit: All instantiations of `SeriesModule` use a string. Would it make more sense to use `SeriesModule<string>` instead?

Reply:
Good call - unfortunately `string` can't be used since (ironically) it's more general than the specification for the parameter allows. I've changed to `RequiredSeriesType` since this matches the specification examples.